.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "wasmCloud",
+    "image": "rust:bullseye",
+    "features": {
+        "devwasm.azurecr.io/dev-wasm/dev-wasm-feature/rust-wasi:0": {},
+        "ghcr.io/lee-orr/rusty-dev-containers/cargo-binstall:latest": {},
```

Comment:
(original: https://github.com/wasmCloud/wasmCloud/pull/335#discussion_r1203946397)

> What does this do?

It installs [Cargo Binstall](https://github.com/cargo-bins/cargo-binstall) - an alternative to cargo install that will download binaries if they exist, and only build from source if binaries aren't found.
This is used by some of the other features to speed up their install process, since they may not need to build if binaries exist.

Reply:
I understand, but I still do not think we should be adding non-standard components here. It's really a rabbit hole where everyone will want to add their favorite piece of tooling here. Let's start small, with something that is minimal and easy to maintain